03-transactions.md

```diff
@@ -417,10 +417,14 @@ Note the reference to the "base fee" for a commitment transaction in the require
 #### Requirements
 
 The fee for an HTLC-timeout transaction:
-  - MUST BE calculated to match:
+  - If `option_zero_fee_second_stage` applies:
+    1. MUST BE 0.
+  - Otherwise, MUST BE calculated to match:
```

Comment:
Should you also update the Requirements in BOLT5 #Generation of HTLC transactions to highlight the enough-fee-contribution with `option_zero_fee_second_stage`. I think it should have been a SHOULD with `option_anchor_output` but now it's obviously a MUST.

Or even make a new requirement to attach a bumping input to second-stage HTLCs, let's avoid zero-fee HTLC transactions ?

Reply:
Added!